### PR TITLE
[Feature][JsonSplit] task mapper and postgre

### PR DESCRIPTION
**Pull request by JinyLeeChina**
task mapper and postgre #4417

### CODE REVIEW

**Comment by chengshiwen** [comment]
@JinyLeeChina Maybe you should pass the **Unit Test** at first

File: sql/dolphinscheduler-postgre.sql [hunk 1]
```diff
@@ -290,30 +290,150 @@ CREATE TABLE t_ds_error_command (
...
+  create_time timestamp DEFAULT NULL ,
   update_time timestamp DEFAULT NULL ,
-  modify_by varchar(36) DEFAULT '' ,
-  resource_ids varchar(64),
   PRIMARY KEY (id),
   CONSTRAINT process_definition_unique UNIQUE (name, project_id)
 ) ;
 
-create index process_definition_index on t_ds_process_definition (project_id,id);
+create index process_definition_index on t_ds_process_definition_log (project_code,id);
```
Comment (lenboo) [inline sql/dolphinscheduler-postgre.sql:340]: i think it should be different with "process_definition_index" in table "t_ds_process_definition_log".

**Review by lenboo — APPROVED** [review]
+1

**Comment by lenboo** [comment]
> @JinyLeeChina Maybe you should pass the **Unit Test** at first

this is a part of the feature. i think he will refactor the ut later.

**Pull request merged**